src/http/structs.rs

```diff
@@ -100,9 +100,24 @@ pub struct EntityPath {
     pub entity: String,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityProfileLink {
+    pub url: String,
+    pub verified: bool,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityProfile {
```

Comment:
is location in the world interesting for a profile?

Reply:
Yes, I'll add it :)